app/wp-content/themes/kstg/library/js/vehicle.js

```diff
@@ -7,6 +7,18 @@ var calloutValue = $('.specialFeature').text();
 
   $(document).ready(function() {
 
+    //function for changing hero image
+
+    //function for highlighting selected image in carousel
+    function setActiveImage(event) {
+      var currentImage = $('ul').find('.borderSelector').toggleClass('borderSelector');
```

Comment:
This isn't quite right. What you're doing makes sense, but you wouldn't assign this to a variable. You could assign just `$('ul').find('.borderSelector')` to a variable (and then use that variable by doing something like `$var.toggleClass..`, if you wanted to, but in this situation there isn't really a need (you would want to do it if you're doing multiple things with that same element so that you don't have to search the DOM every time).